doc/03-Configuration.md

```diff
@@ -0,0 +1,63 @@
+# Configuration
+
+If Icinga Web has been installed but not yet set up, please visit Icinga Web and follow the web-based setup wizard.
+For Icinga Web setups already running, log in to Icinga Web with a privileged user and follow the steps below to
+configure Icinga Notifications Web:
+
+## Module Activation
+
+If you just installed the module, do not forget to activate it on your Icinga 2 Web instance(s) by using your
+preferred way:
+
+- Access the Icinga 2 command-line interface on your master(s) and execute `icingacli module enable notifications`.
+- Navigate to the Icinga 2 website(s), log in as a privileged user and activate the module under `Configuration →
+  Modules → Notifications` by switching the state from `disabled` to `enabled`.
+
+## Database Configuration
+
+Connection configuration for the database `notifications`, which both,
```

Comment:
If you imply a name here by `notifications`, don't? The user can freely choose, this just leaves the user wondering why this term is highlighted.